### ISSUE TITLE
Expose `IonElementLoader`'s functions at top level

### ISSUE THREAD

**Issue by dlurton** [amzn/ion-element-kotlin#18]
[IonElementLoader](https://github.com/amzn/ion-element-kotlin/blob/6859e4436d5aa36ea60ae061d782a53442e1e762/src/com/amazon/ionelement/api/IonElementLoader.kt#L20) is helpful with when dependency injection and mocking is used, but many times we do not use it that way.  We usually just create a single instance which immediately discarded, i.e.:

```
val element = createIonElementLoader(<options>).loadSingleElement("(1)")
```

This is somewhat verbose compared to calling a simple top level function.  It would be great if we could:

```
val element1 = loadSingleElement("(1 2 3)")
val element2 = loadSingleElement("(1 2 3)", nonDefaultLoaderOptions)
```

**Comment by dlurton** [comment]
This is complete on my local dev workstation but is waiting for #22 to be merged.

**Comment by dlurton** [comment]
Implemented in #28
